### ISSUE TITLE
[testmet

### ISSUE THREAD

**Issue by DeckerSU** [KomodoPlatform/komodo#573]
This issue tracker is only for technical issues related to komodod

General Komodo questions and/or support requests and are best directed to [Discord](https://komodoplatform.com/discord)

### Describe the issue
Please provide a general summary of the issue you're experiencing

#### If so, please list the steps to reproduce below:
1. 
2. 
3. 

### Expected behaviour
Tell us what should happen

### Actual behaviour + errors
Tell us what happens instead including any noticable error output (any messages displayed on-screen when e.g. a crash occurred)

### The version of Komodo you were using:
Run `komodod --version` to find out

### Machine specs:
- OS name + version:
- CPU:
- RAM:
- Disk size:
- Disk Type (HD/SDD):
- Linux kernel version (uname -a):
- Compiler version (gcc -version):

### Any extra information that might be useful in the debugging process.
This includes the relevant contents of `~/.komodo/debug.log`. You can paste raw text, attach the file directly in the issue or link to the text via a pastebin type site.
Please also include any non-standard things you did during compilation (extra flags, dependency version changes etc.) if applicable.

### Do you have a backup of `~/.komodo` directory and/or take a VM snapshot?
- Backing up / making a copy of the `~/.komodo` directory might help make the problem reproducible. Please redact appropriately.
- Taking a VM snapshot is really helpful for interactively testing fixes

**Comment by DeckerSU** [comment]
It seems this happened because `testnet` genesis block selected by @jmjatlanta doesn't meet `consensus.powLimit` requirements, like:
`e46e999d6de5ef95427ea51b8242ef7949de22b340b6a484ff154b9991ec9452` is bigger than
`0f0f0f000000000000000000000000000000000000000000000000000000000` and inside `CheckProofOfWork` condition
```
if ( UintToArith256(hash = blkHeader.GetHash()) > bnTarget )
{
...
}
```
leads to fail. The solution is to re-create the `testnet` with new, acceptable genesis block.

**Comment by dimxy** [comment]
yes I suggest we can use the mainnet genesis for testnet (like it was originally)
